appeals/web/src/server/appeals/appeal-details/appellant-case/appellant-case.controller.js

```diff
@@ -230,13 +231,28 @@ export const getAddDocuments = async (request, response) => {
 		return response.status(404).render('app/404.njk');
 	}
 
+	const documentType = currentFolder.path.split('/')[1];
+	let pageHeadingTextOverride;
+
+	switch (documentType) {
```

Comment:
would it be worth breaking this into a utils function and using constants?

Reply:
Yes it would be good to do this at some point, but I don't think this ticket is the right place, ideally it would be done globally as a separate task